dependencies/src/main/java/com/google/cloud/tools/opensource/dependencies/RepositoryUtility.java

```diff
@@ -96,6 +96,21 @@
   private static final ImmutableSet<String> ALLOWED_REPOSITORY_URL_SCHEMES =
       ImmutableSet.of("file", "http", "https");
 
+  public static final DependencySelector nonTestDependencySelector =
```

Comment:
This would be simpler as an outer class.

Reply:
Moved to an independent class.